contractcourt/channel_arbitrator.go

```diff
@@ -802,7 +837,19 @@ func (c *ChannelArbitrator) stateStep(triggerHeight uint32,
 		// If the resolution is empty, then we're done here. We don't
 		// need to launch any resolvers, and can go straight to our
 		// final state.
-		if contractResolutions.IsEmpty() {
+		//
+		// TODO(roasbeef): not needed since we must have a commitment
+		// output in order to make an outgoing HTLC?
```

Comment:
Yes that if we've made an outgoing HTLC, then due to the reserve requirements, we _must_ also have an output since we need funds to make that HTLC. However, thinking about it more: it's possible that we do have enough funds and have met the reserve, but the output is now considered dust.